README.md

```diff
@@ -1,4 +1,4 @@
-# Nu installation example
+# Nu example integration with Apache Iceberg and Nessie catalog
```

Comment:
It'd be great if we could add here a link to the blog (when it's released)

Reply:
Yep, that's the plan